enterprise/glassfish.common/src/org/netbeans/modules/glassfish/common/ProxySettings.java

```diff
@@ -157,7 +157,7 @@ private static String getDefaultUserNonProxyHosts () {
     }
 
     private static String getModifiedNonProxyHosts (String systemPreset) {
-        String fromSystem = systemPreset.replaceAll (";", "|").replaceAll (",", "|"); //NOI18N
+        String fromSystem = systemPreset.replace (";", "|").replace (",", "|"); //NOI18N
         String fromUser = getPresetNonProxyHosts () == null ? "" : getPresetNonProxyHosts ().replaceAll (";", "|").replaceAll (",", "|"); //NOI18N
```

Comment:
FWIW, any particular reason to not replace these occurrence of `replaceAll (";", "|").replaceAll (",", "|")`?

Reply:
Thank you for bringing this up. The initial PR we submitted was not comprehensive. This pull request will be updated with a more comprehensive replacement of replaceAll() -> replace() where applicable.